app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,58 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Create a conversion host and enable it. If you specify a 'queue'
+    # parameter with a true value, the enablement configuration will run as
+    # an MiqTask. Both the 'resource_type' and 'resource_id' are mandatory
+    # arguments, and the 'resource_type' must be either 'Host' or 'VmOrTemplate'.
+    #
+    # You may optionally pass in 'param_v2v_vddk_package_url' or 'auth_user'
+    # arguments as well. The 'auth_user' is only used if the 'queue' parameter
+    # is true.
+    #
+    # POST /api/conversion_hosts {
+    #   "name": "some_name",
+    #   "queue": "true",
+    #   "resource_type": "Host",
+    #   "resource_id": 7
+    #   "param_v2v_vddk_package_url": "some_url"
+    #   "auth_user": "some_user"
+    # }
+    #
+    def create_resource(_type, _id, data)
+      raise BadRequestError, "resource_id must be specified" unless data['resource_id']
+      raise BadRequestError, "resource_type must be specified" unless data['resource_type']
+
+      # Both the 'queue' and 'auth_user' params must be deleted since the model will
+      # otherwise pass the data hash directly as params to ConversionHost.new.
+      queue = data.delete('queue')
+
+      if queue
+        auth_user = data.delete('auth_user')
+        ConversionHost.enable_queue(data, auth_user)
+      else
+        ConversionHost.enable(data)
```

Comment:
Why do we need a non-queue option? This operation involves playbook execution and doesn't sound like a good idea to hold back the anyone in the pipeline.
It's nice to provide options, but since it's not practical for production, we might as well not provide it.

Reply:
I'm just going with what was there when I started. :)

I can alter it so that `enable_queue` is the default, and it could be disabled via `"queue":"false"`. That might be useful for devs, and it's not hard to wrap.